README.md

```diff
@@ -209,6 +209,42 @@ Fivetran offers the ability for you to orchestrate your dbt project through [Fiv
 </details>
 <br>
 
+## (Optional) Step 8: Use predefined Metrics & the dbt Semantic Layer
```

Comment:
> ## (Optional) Step 8: Use predefined Metrics & the dbt Semantic Layer

I think this can just be "Use predefined Metrics" and drop the "& the dbt Semantic Layer" as the querying through the dbt Semantic Layer will happen elsewhere